src/main/java/co/inventorsoft/academy/springBootTask/domain/entity/TournamentBracket.java

```diff
@@ -0,0 +1,25 @@
+package co.inventorsoft.academy.springBootTask.domain.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "tournament")
+@Builder
+@Getter
+@Setter
+@ToString
```

Comment:
Avoid base form of `@ToString` in entity.
If person forgets about this annotation and adds relation, it will produce performance issues.
Explicitly specify what should be included.